<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStu</class>
 <widget class="QDialog" name="addStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加学生</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>录入学生信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name">
       <property name="placeholderText">
        <string>请输入姓名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_number">
       <property name="text">
        <string comment="dsf" extracomment="sdf"/>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>请输入学号</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_male">
       <property name="text">
        <string>男</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_female">
       <property name="text">
        <string>女</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_age">
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4,4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>院系：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_yx">
       <item>
        <property name="text">
         <string>计算机学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>文学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>工商管理学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>理学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>电气工程学院</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>兴趣：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QCheckBox" name="cb_game">
         <property name="text">
          <string>游戏</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="vb_song">
         <property name="text">
          <string>唱歌</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cb_run">
         <property name="text">
          <string>跑步</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cb_ball">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>球类</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,1">
     <item>
      <widget class="QPushButton" name="btn_cancal">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="insGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
